drools-wb-screens/drools-wb-globals-editor/drools-wb-globals-editor-client/src/main/java/org/drools/workbench/screens/globals/client/type/GlobalResourceType.java

```diff
@@ -32,20 +33,30 @@
         extends GlobalResourceTypeDefinition
         implements ClientResourceType {
 
-    private static final Image IMAGE = new Image( GlobalsEditorResources.INSTANCE.images().typeGlobalVariable() );
+    private static final Image IMAGE = new Image(GlobalsEditorResources.INSTANCE.images().typeGlobalVariable());
 
-    @Inject
     private TranslationService translationService;
 
+    public GlobalResourceType(Decision category) {
+        super(category);
+    }
+
+    @Inject
+    public GlobalResourceType(TranslationService translationService,
+                              Decision category) {
```

Comment:
```final```

Reply:
@manstis done all changes, thanks for the review!